### PR TITLE
Restructure rust-geojson, bump to 0.1 

### LINKED ISSUE #3: Implement CRS
http://geojson.org/geojson-spec.html#coordinate-reference-system-objects

### LINKED ISSUE #10: Remove duplicate types: Poly and Polygon
Both represent a Polygon. Poly was added in #8.

### PR DESCRIPTION
**Pull request by frewsxcv**
As discussed in #29

Closes #3

Closes #10

### CODE REVIEW

**Comment by frewsxcv** [comment]
@georust/core Anyone want to look through this? I have a few things I want to do before a new release (add doc comments, examples, auto-generating documentation) so I figure I can do those in a future PR so as to not make this one any bigger.

File: src/feature.rs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        if let Some(ref crs) = feature.crs {
+            map.insert(String::from("crs"), crs.to_json());
+        }
+        if let Some(ref bbox) = feature.bbox {
+            map.insert(String::from("bbox"), bbox.to_json());
+        }
+        if let Some(ref id) = feature.id {
+            map.insert(String::from("id"), id.to_json());
+        }
+        map
```
Comment (mgax) [inline src/feature.rs]: I suggest writing `return map;` for clarity. Leaving out `return` works better for one-liners.

File: src/feature.rs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    }
+}
+
+impl FromObject for Feature {
+    fn from_object(object: &json::Object) -> Result<Self, Error> {
+        Ok(Feature{
+            geometry: try!(object.get_geometry()),
```
Comment (mgax) [inline src/feature.rs]: It feels strange to see utility functions bundled in a trait, they might as well stay separate functions in a module, for example `object.get_geometry()` could be `parse::geometry(object)`, where `parse` is a module that contains all the functions in `ObjectUtils`.

Comment (frewsxcv) [inline src/feature.rs]: A fair criticism. I was just trying it out, I have not decided if I like the pattern or not. I can switch it to a standard function though.

File: src/feature.rs (hunk 1 above)
Comment (frewsxcv) [inline src/feature.rs]: Sure, I can add them in. I will do that later today.

Comment (frewsxcv) [inline src/feature.rs]: https://github.com/georust/rust-geojson/commit/bf4db74124ff1a62a1e4d49c82343d3cc89457a4

File: src/feature.rs (hunk 2 above)
Comment (frewsxcv) [inline src/feature.rs]: https://github.com/georust/rust-geojson/commit/98de07143dc55356b6f034981471371320227c14

**Comment by frewsxcv** [comment]
@mgax Let me know if you see anything else

**Comment by mgax** [comment]
:+1: lgtm

**Comment by skj3gg** [comment]
+1

**Pull request merged**